firebase-common/src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -525,7 +525,7 @@ protected FirebaseApp(Context applicationContext, String name, FirebaseOptions o
     idTokenListenersCountChangedListener = new DefaultIdTokenListenersCountChangedListener();
 
     sharedPreferences =
-        applicationContext.getSharedPreferences(FIREBASE_APP_PREFS, Context.MODE_PRIVATE);
+        applicationContext.getSharedPreferences(getSharedPrefsName(name), Context.MODE_PRIVATE);
```

Comment:
What is the state of this rollout?
Is it going to continue working for apps that may pick up this update, but may have already set this bit to a different location?

Reply:
It's not yet rolled out/documented/supported, so this should not break existing apps.